crates/db_views/src/structs.rs

```diff
@@ -133,7 +134,7 @@ pub struct CustomEmojiViewForm {
   pub id: i32,
   pub category: String,
   pub shortcode: String,
-  pub image_url: String,
+  pub image_url: Url,
```

Comment:
Should use `DbUrl` here. But tbh if this compiles and works, im not sure why we needed the separate DbUrl type at all.